Makefile

```
@@ -8,6 +8,8 @@ PYTHONIOENCODING=utf8
 
 # Docker container tag ("$(DOCKER_TAG)")
 DOCKER_TAG = 'ocrd/kraken'
+DOCKER_BASE_IMAGE = docker.io/ocrd/core-cuda:v2.70.0
```

I have not tested this recently, but it should give the benefit of installing the CUDA variant of Pytorch instead of the CPU variant:

```suggestion
DOCKER_BASE_IMAGE = docker.io/ocrd/core-cuda-torch:v2.70.0
```